k8s.gcr.io/images/k8s-staging-cluster-api-azure/OWNERS

```diff
@@ -6,11 +6,11 @@ approvers:
 - CecileRobertMichon
 - devigned
 - nader-ziada
+- shysank
 reviewers:
 - cpanato
 - juan-lee
 - mboersma
```

Comment:
can you also please add @jackfrancis and @jsturtevant so this is up to date with our owners file?

Reply:
done